docs/var.tfvars-doc.md

```diff
@@ -183,6 +183,8 @@ Edit it as per your requirements.
 cluster_domain              = "ibm.com"
 cluster_id_prefix           = "test-ocp"
 cluster_id                  = ""
+
+use_zone_info_for_instance  = true
```

Comment:
The variable name mentions only about instance. We are also using it for network, keypair, volume name as well.
Suggestion for name: use_zone_for_names

Reply:
Done.

Reply:
@cs-zhang don't see the latest changes.. Did you push it ?